### ISSUE TITLE
请教一下：后期会支持其他层的量化么？

### ISSUE THREAD

**Issue by hbxtxjk** [aovoc/nnieqat-pytorch#9]
感谢分享！
调试代码，发现quantize.py文件只支持Convolution层量化，没有对Deconvolution、DepthwiseConv、InnerProduct等层进行实现，如果我在Deconvolution层后加入BN层，训练时则会在conv_module = _fuse_conv_bn(conv_module, child)这句传入conv_module=None的错误参数，虽然通过调整(修改为 if isinstance(child, (torch.nn.BatchNorm1d, torch.nn.BatchNorm2d, torch.nn.BatchNorm3d)) and (conv_module is not None): )能过滤掉，但是属于治标不治本的做法，所以好奇问问：后期会有其他层实现吗？

**Comment by aovoc** [comment]
https://github.com/aovoc/nnieqat-pytorch/blob/be2095c6e4c9b663b08437c052b00e9bcded8602/nnieqat/quantize.py#L345 
权重量化实现是对除bn外所有的weight进行量化，Deconvolution 包含weight, DepthwiseConv 在pytorch下对Convolution层设置下 group是可以的，caffe的InnerProduct层对应pytorch的linear 层也是没问题的

**Comment by aovoc** [comment]
conv 和bn的合并问题已解决
https://github.com/aovoc/nnieqat-pytorch/blob/be2095c6e4c9b663b08437c052b00e9bcded8602/nnieqat/quantize.py#L174

**Comment by hbxtxjk** [comment]
很抱歉现在才看到，谢谢！
